src/main/java/com/infomaniak/lib/core/MatomoCore.kt

```diff
@@ -33,10 +33,11 @@ interface MatomoCore {
     val Context.tracker: Tracker
     val siteId: Int
 
-    fun Context.buildTracker(): Tracker {
+    fun Context.buildTracker(isOptOut: Boolean): Tracker {
```

Comment:
Below you used the name "shouldOptOut" as the argument name, we could use the same here